ibis/pandas/execution/generic.py

```diff
@@ -768,6 +768,14 @@ def execute_union_dataframe_dataframe(op, left, right, distinct, **kwargs):
     return result.drop_duplicates() if distinct else result
 
 
+@execute_node.register(ops.Intersection, pd.DataFrame, pd.DataFrame)
+def execute_intersection_dataframe_dataframe(
+    op, left: pd.DataFrame, right, **kwargs
+):
+    result = left.merge(right, on=list(left.columns))
```

Comment:
should these be an outer merge (as is the default)?

Reply:
Right here in the docs it says the default is inner, https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.merge.html